pandas/_libs/hashing.pyx

```diff
@@ -21,8 +21,10 @@ import_array()
 
 from pandas._libs.util cimport is_nan
 
-DEF cROUNDS = 2
-DEF dROUNDS = 4
+
+cdef enum:
```

Comment:
Small nit: why is `enum` used here? - `cROUNDS` and `dROUNDS` just look like unrelated globals based on the usage below. While equivalent, maybe clearer to use an explicit `int` type?